src/it/java/teammates/it/storage/sqlapi/NotificationDbIT.java

```diff
@@ -38,4 +38,31 @@ public void testCreateNotification() throws EntityAlreadyExistsException, Invali
         Notification actualNotification = notificationsDb.getNotification(notificationId);
         verifyEquals(newNotification, actualNotification);
     }
+
+    @Test
+    public void testGetNotification() throws EntityAlreadyExistsException, InvalidParametersException {
+        ______TS("success: get a notification that already exists");
+        Notification newNotification = new Notification.NotificationBuilder()
+                .withStartTime(Instant.parse("2011-01-01T00:00:00Z"))
+                .withEndTime(Instant.parse("2099-01-01T00:00:00Z"))
+                .withStyle(NotificationStyle.DANGER)
+                .withTitle("A deprecation note")
+                .withTargetUser(NotificationTargetUser.GENERAL)
+                .withMessage("<p>Deprecation happens in three minutes</p>")
+                .build();
+
+        notificationsDb.createNotification(newNotification);
+
+        UUID notificationId = newNotification.getNotificationId();
+        Notification actualNotification = notificationsDb.getNotification(notificationId);
+        verifyEquals(newNotification, actualNotification);
+
+        ______TS("success: get a notification that does not exist");
+        UUID nonExistentId = UUID.randomUUID();
+        while (nonExistentId.equals(notificationId)) {
+            nonExistentId = UUID.randomUUID();
+        }
```

Comment:
Can we move it into a method in the parent class. Maybe generateDifferentUUID(UUID);.